core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -1786,6 +1814,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 CHECK_MEMORY_OVERFLOW(4);
                 PUSH_I32(LOAD_I32(maddr));
                 (void)flags;
+                CHECK_READ_WATCHPOINT(addr, offset);
```

Comment:
How about moving this line before line 1816? Normally we put `(void)xxx` at the end.

Reply:
Yes, it looks better :)